mitmproxy/tools/cmdline.py

```diff
@@ -568,23 +568,23 @@ def proxy_ssl_options(parser):
 def onboarding_app(parser):
     group = parser.add_argument_group("Onboarding App")
     group.add_argument(
-        "--noapp",
-        action="store_false", dest="app",
+        "--no-onboarding",
+        action="store_false", dest="onboarding",
         help="Disable the mitmproxy onboarding app."
```

Comment:
Is everybody familiar with the onboarding app in general?
Maybe We should add a one sentence description of what the purpose is?